pbj-core/pbj-runtime/src/jmh/java/com/hedera/pbj/runtime/io/ByteBufferGetByte.java

```diff
@@ -68,14 +68,14 @@ public void directBufferGet(final Blackhole blackhole) {
     @Benchmark
     public void heapUnsafeGet(final Blackhole blackhole) {
         for (int i = 0; i < size; i++) {
-            blackhole.consume(UnsafeUtils.getByteHeap(heapBuffer, i));
+            blackhole.consume(UnsafeUtils.getHeapBufferByte(heapBuffer, i));
         }
     }
 
     @Benchmark
     public void directUnsafeGet(final Blackhole blackhole) {
         for (int i = 0; i < size; i++) {
-            blackhole.consume(UnsafeUtils.getByteDirect(directBuffer, i));
+            blackhole.consume(UnsafeUtils.getDirectBufferByte(directBuffer, i));
```

Comment:
If "buffer" and "byte" are swapped, it would be `getDirectByteBuffer()`, which is totally wrong. This method gets a byte, not a buffer. There is also a method called `getHeapBufferByte()` in `UnsafeUtils`, and also `getArrayByte()` to get a byte from an array.

Reply:
Oh, I should've read the implementation of the method. This looks good then. Thanks for correcting me.